nodes/usb_cam_node.cpp

```diff
@@ -105,7 +105,7 @@ class UsbCamNode
     node_.param("autofocus", autofocus_, false);
     node_.param("focus", focus_, -1); //0-255, -1 "leave alone"
     // enable/disable autoexposure
-    node_.param("autoexposure", autoexposure_, true);
+    node_.param("autoexposure", autoexposure_, false);
```

Comment:
What's the rationale for disabling auto-exposure by default? It seems like a useful feature for the most common use case.

Reply:
It can affect the framerate st by people. see https://answers.ros.org/question/227891/usb_cam-camera-frame-rate-and-rostopic/